features/steps/app_steps.rb

```diff
@@ -10,15 +10,17 @@
 end
 
 When("I run the configured scenario and relaunch the crashed app") do
-  platform = Maze::Helper.get_current_platform
-  case platform
+  case Maze::Helper.get_current_platform
   when 'ios'
     run_and_relaunch
   when 'macos'
     $scenario_mode = $last_scenario[:scenario_mode]
     execute_command($last_scenario[:action], $last_scenario[:scenario_name])
+  when 'watchos'
+    run_watchos_app
+    sleep 10 # we don't have a way to check if the app is still running
   else
-    raise "Unknown platform: #{platform}"
+    raise 'Unsupported platform'
```

Comment:
Any reason to remove the printing of `platform`?  Couldn't that be a bit annoying for someone if it falls into this branch?

Reply:
Seemed unlikely someone wouldn't know which platform they're running tests on... but I can restore this

Reply:
For me it was the fact that you're at the mercy of what `Maze::Helper.get_current_platform` returns.  It's not necessarily just what you pass in on the command line.